integration-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Start a SPIRE Server and Agent and run the integration tests
+# Only works on Linux.
+
+set -euf -o pipefail
+
+spire_folder="spire-858d04b"
+
+function cleanup() {
+  killall -9 spire-agent || true
+  killall -9 spire-server || true
+  rm -f /tmp/spire-server/private/api.sock
+  rm -f /tmp/spire-agent/public/api.sock
+  rm -rf ${spire_folder}
+}
+
+# Some cleanup: kill spire processes that could have remained from previous run
+cleanup
+
+# Install and run a SPIRE server
+curl -s -N -L https://github.com/spiffe/spire/releases/download/v1.0.0/spire-1.0.0-linux-x86_64-glibc.tar.gz | tar xz
+pushd "${spire_folder}"
+bin/spire-server run -config conf/server/server.conf &
+sleep 10
```

Comment:
Relying on sleeps is brittle. I would suggest doing a regular health check on the server instead:
```bash
spire_server_started=0
for i in {1..10}
do
    if bin/spire-server healthcheck -socketPath ${spire_server_socket_path} >/dev/null 2>&1; then
        spire_server_started=1
        break
    fi

    sleep 1
done

if [ ${spire_server_started} -ne 1 ]; then
    cat ${spire_server_log_file} >&2
    echo 'SPIRE Server failed to start' >&2
    exit 1
fi
```